### PR TITLE
Infodev-2366 live chat support audit

### PR DESCRIPTION
**Pull request by chri2547**

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-9852e05f7a/how-to/)

**Comment by cyrichardson** [comment]
@chri2547, on it!

File: content/cloud-servers/getting-fanatical-support-7-live-chat.md [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-last_modified_by: Stephanie Fillmon
+last_modified_date: '2018-08-30'
+last_modified_by: Chris Moyer
 product: Cloud Servers
 product_url: cloud-servers
 ---
 
-Rackspace has Support Technicians standing by 24 hours a day to answer any questions you have. You can find a link to Live Chat on the [Rackspace homepage](https://www.rackspace.com) as a expanding tab on the right-hand side of the page. Under the Live Chat link, you will be directed into a Live Chat session with specialists
-trained to support our various products. Choose the appropriate Live Chat link for the product you need assistance with.
+Rackspace Support Technicians are available 24 hours a day to answer any questions that you have.
```
Comment (cyrichardson) [inline content/cloud-servers/getting-fanatical-support-7-live-chat.md]: technicians

**Comment by cyrichardson** [comment]
@chri2547 Only 1 suggestion so I provided it inline.

File: content/cloud-servers/getting-fanatical-support-7-live-chat.md [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
+
+After you click the **Live Chat** link at the bottom of the [Rackspace homepage](https://www.rackspace.com), you interact with a specialist trained to support our products.
 
 ### Additional resources
```
Comment (cyrichardson) [inline content/cloud-servers/getting-fanatical-support-7-live-chat.md]: @chri2547 It looks like there last 2 links are the same. Will need to delete one of them. Also, please change "Getting Started" in the link to "Getting started".

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b734507209/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9361617145/how-to/)

File: content/cloud-servers/getting-fanatical-support-7-live-chat.md [hunk 3]
```diff
@@ -1,23 +1,23 @@
 ---
 permalink: getting-fanatical-support-7-live-chat/
-audit_date:
+audit_date: 2018-08-30'
```
Comment (RackerKate) [inline content/cloud-servers/getting-fanatical-support-7-live-chat.md]: Need an opening quote mark here.

File: content/cloud-servers/getting-fanatical-support-7-live-chat.md [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
+last_modified_by: Chris Moyer
 product: Cloud Servers
 product_url: cloud-servers
 ---
 
-Rackspace has Support Technicians standing by 24 hours a day to answer any questions you have. You can find a link to Live Chat on the [Rackspace homepage](https://www.rackspace.com) as a expanding tab on the right-hand side of the page. Under the Live Chat link, you will be directed into a Live Chat session with specialists
-trained to support our various products. Choose the appropriate Live Chat link for the product you need assistance with.
+Rackspace Support technicians are available 24 hours a day to answer any questions that you have.
+
+After you click the **Live Chat** link at the bottom of the [Rackspace homepage](https://www.rackspace.com),
```
Comment (RackerKate) [inline content/cloud-servers/getting-fanatical-support-7-live-chat.md]: Home page is 2 words per Style Guide.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-8d70b950ac/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-14add7ab65/how-to/).

**Pull request merged**